src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -1086,6 +1102,17 @@ private void ThrowIfClosed()
             }
         }
 
+        private void ThrowIfMigrationInProgress()
+        {
+            bool rejectWriteState = this.actorService.GetRejectWriteState();

```

Comment:
The check has to be (Is_KVS_Migration_Source and RejectWrites_Set) or (Is_RC_MigrationTarget and MigrationPhase_Is_Not_Completed) instead?

Reply:
Already doing this check in Microsoft.ServiceFabric.Actors.Migration.Utility.GetRejectWriteState.